manifests/m/Microsoft/SaRA/17.01.0268.015/Microsoft.SaRA.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Microsoft.SaRA
+PackageVersion: 17.01.0268.015
+PackageLocale: en-US
+Publisher: Microsoft
+PackageName: SaRA

```

```suggestion
PackageName: Microsoft Support and Recovery Assistant
```
This should match control panel